README.md

```diff
@@ -1,27 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TV Shows Hub
 
-> Description the project.
+> This project requires us to build a web application using external APIs. We have selected an API that provides data about TV shows and have built the web app around it.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML/CSS and JavaScript
+- Frameworks: Node.js 
+- APIs: IMDb API and Involvement API
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo]()
+[Vide Demo](./video/JSCapstone.mp4)
 
```

Comment:
- [ ] The video should contains a highlight about a piece of the code that you find interesting or you are proud of making.